packages/modular-scripts/src/cli.ts

```diff
@@ -41,13 +41,15 @@ program
     'Equivalent of --prefer-offline for yarn installations',
     true,
   )
+  .option('--verbose', 'Run yarn commands with --verbose set')
```

Comment:
We should set `process.env.MODULAR_LOGGER_DEBUG` to true if they want verbose logs.

Reply:
I've updated `MODULAR_LOGGER_DEBUG` to default to `process.argv.includes('--verbose') to match `create-modular-react-app`